pcsx2/Docs/GameIndex.md

```diff
@@ -184,12 +207,12 @@ These values are in a key-value format, where the value is assumed to be an inte
 *   `mvuFlagSpeedHack`
 *   Accepted Values - `0` / `1`
 *   Katamari Damacy have weird speed bug when this speed hack is enabled (and it is by default)
-
-<!-- [list-item-spacing] Missing new line after list item->
-
--   `InstantVU1SpeedHack`
--   Accepted Values - `0` / `1`
--   Games such as Parappa the Rapper 2 need VU1 to sync, so you can force disable the speedhack here
+*   `MTVUSpeedHack`
+*   Accepted Values - `0` / `1`
+*   T-bit games dislikes MTVU
+*   `InstantVU1SpeedHack`
+*   Accepted Values - `0` / `1`
+*   Games such as Parappa the Rapper 2 need VU1 to sync, so you can force disable the speedhack here
```

Comment:
Period at end. and "with this parameter" is better than "disable the speedhack here" because nobody is disabling it "here"